app/src/main/res/drawable/launch_screen.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Nextcloud Talk application
+  ~
+  ~ Copyright (C) 2018 Edvard Holst
```

Comment:
Andy...

Reply:
Nope Edvart aka @ardevd, he did this implementation for the files client and I copied it here, to it is still his copyright 👍